pacmacs.el

```diff
@@ -130,18 +131,58 @@
         :column column
         :type 'wall))
 
-(defun pacmacs--make-pill (row column anim-name points)
+(defun pacmacs--make-pill (row column anim-name points type)
```

Comment:
Well, probably it would be good to create to separate functions for big and regular pills. I'll take a look into that anyway. Thanks!